src/components/Sidebar/Properties/PropertyOwnerHeader.jsx

```diff
@@ -115,31 +127,19 @@ const mapStateToProps = (state, ownProps) => {
   } else if (state.propertyTree.properties[uri + '.Renderable.Enabled']) {
     quickToggleUri = uri + '.Renderable.Enabled';
   }
-  //check for layers so we can make the titles green
-  var isLayer = false;
-  LayerGroupKeys.forEach( (layerGroup) => {
-    if ( (uri.indexOf(layerGroup) > -1) && !(uri.endsWith(layerGroup)) ) {
-      isLayer = true;
-    }
-  });
-  let isHeightLayer = quickToggleUri && (quickToggleUri.lastIndexOf('HeightLayers') > 0);
-  var titleClass = "";
-  var enabled = quickToggleUri && state.propertyTree.properties[quickToggleUri].value;
-  if (isLayer) {
-      if (enabled) {
-        titleClass = styles.greenTitle;
-      } else if (isHeightLayer) {
-        //make height layers visually distinguishable from other layers
-        //specificlly color layers where they might have the same name
-        titleClass = styles.grayTitle;
```

Comment:
did we loose this functionality as part of this? The idea being height layers have some visual difference then other layers.

Reply:
Yes. I didn't see why height layers should be treated differently than other layers as they are under two visual groups in the GUI anyways. Why should disabled height layers be shown as gray, but not any other layers?

However, if you have some motivation for it, I can bring it back

Reply:
I guess the motivation might be that height layers don't necessarily lead to a visual change on the globe unless you're really close to it 🤔